### PR TITLE
[12.0][MIG] purchase_product_usage

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

File: purchase_product_usage/views/purchase_order_line_view.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <field name="inherit_id" ref="purchase.purchase_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_line']/tree/field[@name='name']"
+                   position="after">
+                <field name="usage_id"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="purchase_order_form_form" model="ir.ui.view">
```
Comment (tbaden) [inline purchase_product_usage/views/purchase_order_line_view.xml]: why are there two records for the same inherited view (purchase.purchase_order_form)?
 <record id="purchase_order_form"
and
<record id="purchase_order_form_form"

**Review by tbaden — COMMENTED** [review]
thanks for the work 👍 
I did a code review:
I got one question (see below)

File: purchase_product_usage/views/purchase_order_line_view.xml (hunk 1 above)
Comment (MiquelRForgeFlow) [reply purchase_product_usage/views/purchase_order_line_view.xml]: solved

**Review by AdriaGForgeFlow — APPROVED** [review]
Tested on runbot :+1:

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**